src/js/game/LevelMVC/LevelBlock.js

```diff
@@ -323,9 +324,11 @@ module.exports = class LevelBlock {
    * @return {boolean}
    */
   static isFlat(blockType) {
-    return blockType.substring(0, 5) === "rails" ||
+    // Starts With 'rail' will capture rails and railsRedstoneTorch.
+    return blockType.isRail ||
         blockType.startsWith("redstoneWire") ||
-        blockType.startsWith("pressurePlate");
+        blockType.startsWith("pressurePlate") ||
+        blockType === "torch";
```

Comment:
we still don't want torches to be `flat`

Reply:
ack, I put that before our conversation and hard just forgot to take it out.